src/amo/utils/reviewRootDir.ts

```diff
@@ -0,0 +1,44 @@
+import * as fs from "fs";
+import * as vscode from "vscode";
+
+export async function selectRootFolder() {
+  const options: vscode.OpenDialogOptions = {
+    canSelectFiles: false,
+    canSelectFolders: true,
+    canSelectMany: false,
+    openLabel: "Select Addon Review Workspace",
+  };
+
+  const selectedFolders = await vscode.window.showOpenDialog(options);
+  if (selectedFolders && selectedFolders.length > 0) {
+    const rootFolder = selectedFolders[0].fsPath;
+    return rootFolder;
```

Comment:
nit: you can just return `selectedFolders[0].fsPath` without having to create a const